examples/is13.rs

```diff
@@ -0,0 +1,45 @@
+// Example "is-thirteen" is a useless(almost) program for just checking if a number is equal to 13.
+//
+// But it's also important because it shows how to compile a program from C source code and then run
+// it by ckb-vm. So, Let's start.
+//
+// To build the "is13.c" to riscv output, you will need to install "riscv-gnu-toolchain" in your system.
+//
+//   $ git clone --recursive https://github.com/riscv/riscv-gnu-toolchain
+//   $ cd riscv-gnu-toolchain
+//   $ mkdir build && cd build
+//   $ ../configure --prefix=/opt/riscv --with-arch=rv32imac --with-abi=ilp32
+//   $ make
+//
+// On my ubuntu machine, it takes 1 hours. Sad.
+//
+// Then, you can some c codes like is13.c, and build it by "riscv32-unknown-elf-gcc"
+//
+//   $ riscv32-unknown-elf-gcc -o is13 is13.c
```

Comment:
I think we don't need to commit the binary file into the repository since it's an example to let the user go through the whole process.

Reply:
Removed it. It should be done by the user himself, emmm.

Reply:
Please add it to `.gitignore` as well

Reply:
Added!